src/Framework/DataProviderTestSuite.php

```diff
@@ -19,17 +18,16 @@
 final class DataProviderTestSuite extends TestSuite
 {
     /**
-     * @var array<string>
+     * @var array<ExecutionOrderDependency>
```

Comment:
psalm supports `list<ExecutionOrderDependency>` which makes the user aware that this is not an ordered list

Reply:
👍 for using `list<>` whereever it makes sense / is possible.

Reply:
okidoki! will use that from now on :)